catalog/spatial/kml/spatial-kml-transformer/src/main/java/org/codice/ddf/spatial/kml/transformer/HandlebarsMetacard.java

```diff
@@ -78,6 +78,19 @@ public AttributeFormat setValue(AttributeFormat value) {
         public int compareTo(AttributeEntry other) {
             return this.getKey().getName().compareTo(other.getKey().getName());
         }
+        @Override
+        public boolean equals(Object other) {
```

Comment:
Don't we care about the `value` field for equality?

We should reformat this method too. Format seems to be off.